src/fal/planner/plan.py

```diff
@@ -77,3 +77,26 @@ def plan_graph(graph: nx.DiGraph) -> nx.DiGraph:
         _reduce_subgraph(new_graph, nodes)
 
     return new_graph
+
+
+# TEMPORARY!!!
+def __test_reorder_graph(graph: nx.DiGraph) -> nx.DiGraph:
```

Comment:
is this to be able to run integration tests? little confused about why you need this vs. writing unit tests that dont have any before or after scripts as inputs

Reply:
> is this to be able to run integration tests? little confused about why you need this vs. writing unit tests that dont have any before or after scripts as inputs

True, that is the plan for the future. I just added this to temporarily verify the execution of DBT models etc. works. Can remove it though. We currently don't have any automated tests for the parallelization, so it is for local development only.